### DISCUSSION TITLE
how to change wwwroot path, when debug  webassembly project no server host

### DISCUSSION THREAD

**Discussion by srxqds** [dotnet/aspnetcore#59132] [General]
how to change wwwroot path, when debug  webassembly project no server it host by visual sudio ide?

**Comment by tossnet** [comment]
Are you talking about the <base href="/" /> tag ? Wouldn't an approch using `@inject IWebHostEnvironment Env`  and `@if (Env.IsDevelopment()) {} else {} `work ?

**Comment by srxqds** [comment]
but we only dotnet wasm client project no server host project

**Comment by tossnet** [comment]
ah sorry yes. But I think I've got a good lead for you:  [Read the environment client-side in a Blazor Web App](https://learn.microsoft.com/en-us/aspnet/core/blazor/fundamentals/environments?WT.mc_id=DT-MVP-5000641&view=aspnetcore-9.0#read-the-environment-client-side-in-a-blazor-web-app)
